src/main/java/appeng/util/ReadableNumberConverter.java

```diff
@@ -23,7 +23,7 @@ public enum ReadableNumberConverter implements ISlimReadableNumberConverter, IWi
     /**
      * String representation of the sorted postfixes
      */
-    private static final char[] ENCODED_POSTFIXES = "KMGTPE".toCharArray();
+    private static final char[] ENCODED_POSTFIXES = "KMBTQU".toCharArray();
```

Comment:
You can't just get rid of the metric prefixes like that

Reply:
Agreed, this is a travesty!